src/RadhydroShell/test_radhydro_shell.cpp

```diff
@@ -232,6 +232,10 @@ void RadhydroSimulation<ShellProblem>::setInitialConditionsAtLevel(int lev) {
       state(i, j, k, RadSystem<ShellProblem>::x1RadFlux_index) = Frad_xyz;
       state(i, j, k, RadSystem<ShellProblem>::x2RadFlux_index) = Frad_xyz;
       state(i, j, k, RadSystem<ShellProblem>::x3RadFlux_index) = Frad_xyz;
+
+      state(i, j, k, RadSystem<ShellProblem>::gasEnergy_index) = 0.;
```

Comment:
This is not correct. The gas density and energy are set a few lines above in this file. They are not zero.

Reply:
On lines 236 and 237 the gas energy and density are being set to zero, no? Namely:
state(i, j, k, RadSystem<ShellProblem>::gasEnergy_index) = 0.;
state(i, j, k, RadSystem<ShellProblem>::gasDensity_index) = 0.;

Reply:
Those are lines you added, right?

The density is set on line https://github.com/BenWibking/quokka/blob/e6391c605b55afbce73f194c86114bb98a0e54e9/src/RadhydroShell/test_radhydro_shell.cpp#L224

Reply:
The total gas energy is set here:
https://github.com/BenWibking/quokka/blob/e6391c605b55afbce73f194c86114bb98a0e54e9/src/RadhydroShell/test_radhydro_shell.cpp#L228

Reply:
Fixed.